packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.tsx

```diff
@@ -0,0 +1,696 @@
+import * as React from 'react';
+import {
+  autobind,
+  BaseComponent,
+  css,
+  findIndex,
+  getId
+} from '../../Utilities';
+import {
+  ISwatchColorPickerProps,
+  ISwatchColorPickerItemProps,
+  SwatchColorPickerItemType
+} from './SwatchColorPicker.Props';
+import { DirectionalHint } from '../../ContextualMenu';
+import { getColorFromString } from '../../utilities/color/colors';
+import { Grid } from '../../utilities/grid/Grid';
+import { DefaultButton, CommandButton, IButtonProps } from '../../Button';
+import { Callout } from '../../Callout';
+import { FocusZone } from '../../FocusZone';
+import * as stylesImport from './SwatchColorPicker.scss';
+const styles: any = stylesImport;
+
+interface ISwatchColorPickerOptionProps {
+  item: ISwatchColorPickerItemProps;
+  id: string;
+  posInSet?: number;
+  setSize?: number;
+  disabled?: boolean;
+  selectedIndex?: number;
+  onClick: (item: ISwatchColorPickerItemProps) => void;
+  onHover?: (id?: string, color?: string) => void;
+  onFocus?: (id?: string, color?: string) => void;
+  role?: string;
+  className?: string;
+  cellShape?: string;
+}
+
+const SwatchColorPickerOption: React.StatelessComponent<ISwatchColorPickerOptionProps> =
+  (props: ISwatchColorPickerOptionProps) => {
+    let {
+      item,
+      id,
+      posInSet,
+      setSize,
+      className,
+      role,
+      selectedIndex,
+      disabled,
+      cellShape,
+      onClick,
+      onHover,
+      onFocus
+    } = props;
+    return (
+      <CommandButton
+        { ...item.menuItemButtonProps }
+        id={ id + '-item' + item.index }
+        data-index={ item.index }
+        data-is-focusable={ true }
+        aria-posinset={ posInSet && posInSet }
+        aria-setsize={ setSize && setSize }
+        disabled={ disabled }
+        className={ css(className,
+          {
+            ['is-selected ' + styles.cellIsSelected]: (selectedIndex && selectedIndex === item.index),
+            ['is-disabled ' + styles.disabled]: disabled
+          }
+        ) }
+        onClick={ _onClick }
+        onMouseEnter={ _onMouseEnter }
+        onMouseLeave={ _onMouseLeave }
+        onFocus={ _onFocus }
+        role={ role }
+        aria-selected={ selectedIndex && selectedIndex === item.index ? 'true' : 'false' }
```

Comment:
can this just be (selectedIndex === item.index).toString(); or new Boolean(selectedIndex === item.index)